docs/content.zh/docs/ops/state/checkpoints_vs_savepoints.md

```diff
@@ -24,72 +24,65 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Checkpoints vs. Savepoints
+<a name="checkpoints-vs-savepoints"></a>
 
-## Overview
+# Checkpoints 与 Savepoints
 
-Conceptually, Flink's [savepoints]({{< ref "docs/ops/state/savepoints" >}}) are different from [checkpoints]({{< ref "docs/ops/state/checkpoints" >}}) 
-in a way that's analogous to how backups are different from recovery logs in traditional database systems.
+<a name="overview"></a>
 
-The primary purpose of checkpoints is to provide a recovery mechanism in case of unexpected job failures. 
-A [checkpoint's lifecycle]({{< ref "docs/dev/datastream/fault-tolerance/checkpointing" >}}) is managed by Flink, 
-i.e. a checkpoint is created, owned, and released by Flink - without user interaction. 
-Because checkpoints are being triggered often, and are relied upon for failure recovery, the two main design goals for the checkpoint implementation are 
-i) being as lightweight to create and ii) being as fast to restore from as possible. 
-Optimizations towards those goals can exploit certain properties, e.g., that the job code doesn't change between the execution attempts. 
+## 概述
+
+从概念上讲，Flink 的 [savepoints]({{< ref "docs/ops/state/savepoints" >}}) 与 [checkpoints]({{< ref "docs/ops/state/checkpoints" >}}) 的不同之处类似于传统数据库系统中的备份与恢复日志之间的差异。
+
+Checkpoints 的主要目的是为意外失败的作业提供恢复机制。
+[Checkpoint 的生命周期]({{< ref "docs/dev/datastream/fault-tolerance/checkpointing" >}}) 由 Flink 管理，
+即 Flink 创建，管理和删除 checkpoint - 无需用户交互。
+作为一种恢复和定期触发的方法，Checkpoint 实现有两个设计目标：i）轻量级创建和 ii）尽可能快地恢复。 可能会利用某些特定的属性来达到这个，例如， 工作代码在执行尝试之间不会改变。
 
 {{< hint info >}}
-- Checkpoints are automatically deleted if the application is terminated by the user 
-(except if checkpoints are explicitly configured to be retained).
-- Checkpoints are stored in state backend-specific (native) data format (may be incremental depending on the specific backend).
+- 在用户终止作业后，会自动删除 Checkpoint（除非明确配置为保留的 Checkpoint）。
+- Checkpoint 以状态后端特定的（原生的）数据格式存储（根据特定后端可能是增量的）。
 {{< /hint >}}
 
-Although [savepoints]({{< ref "docs/ops/state/savepoints" >}}) are created internally with the same mechanisms as
-checkpoints, they are conceptually different and can be a bit more expensive to produce and restore from. Their design focuses
-more on portability and operational flexibility, especially with respect to changes to the job. 
-The use case for savepoints is for planned, manual operations. For example, this could be an update of your Flink version, changing your job graph, and so on.
+尽管 [savepoints]({{< ref "docs/ops/state/savepoints" >}}) 在内部使用与 checkpoints 相同的机制创建，但它们在概念上有所不同，并且生成和恢复的成本可能会更高一些。他们的设计更侧重于可移植性和操作灵活性，尤其是在 job 变更方面。Savepoint 的用例是计划中的、手动的运维。例如，可能是你的 Flink 版本更新，更改你的作业图等等。
```

Comment:
```suggestion
尽管 [savepoints]({{< ref "docs/ops/state/savepoints" >}}) 在内部使用与 checkpoints 相同的机制创建，但它们在概念上有所不同，并且生成和恢复的成本可能会更高一些。Savepoints的设计更侧重于可移植性和操作灵活性，尤其是在 job 变更方面。Savepoint 的用例是针对计划中的、手动的运维。例如，可能是更新你的 Flink 版本，更改你的作业图等等。
```

Reply:
Thanks @JingGe , much clear with your suggestion ! committed it.